### ISSUE TITLE
Scorer awarded start gate and distance beyond minimum.

### ISSUE THREAD

**Issue by philderbeast** [FAI-CIVL/FAI-Airscore#232]
For Forbes 2018, task 7, the scorer gave Rick Martin a start gate and a distance well beyond minimum distance. When I import this comp, leave it as external and then view the results he's not given his due start time or distance and scores zero. FS scores him 158 points. The scorer also penalised him 70 points for starting early.

![Screen Shot 2020-12-26 at 9 04 53 PM](https://user-images.githubusercontent.com/633283/103162421-7065a500-47be-11eb-8976-e259c3496b17.png)

**Comment by philderbeast** [comment]
Relevant sections of the `*.fsdb`:

```xml
<!-- participants snippet -->
<FsParticipant id="85" name="Rick Martin" />
<!-- task 7 results snippet -->
<FsParticipant id="85">
  <FsFlightData distance="95.030" bonus_distance="95.030" started_ss="2018-01-05T13:40:00+11:00"
    finished_ss="" altitude_at_ess="0" finished_task="" tracklog_filename="" lc="0" iv="0"
    ts="2018-01-05T22:20:04+11:00" alt="0" bonus_alt="0" max_alt="0"
    last_tracklog_point_distance="<large -ve number>"
    bonus_last_tracklog_point_distance="<large -ve number>"
    last_tracklog_point_time="" last_tracklog_point_alt="0" />
  <FsResultPenalty penalty="0" penalty_points="70" penalty_reason="Early Start" />
  <FsResult rank="35" points="158" distance="95.03" ss_time="00:00:00" finished_ss_rank="26"
    distance_points="227.9" linear_distance_points="106.32" difficulty_distance_points="121.6" time_points="0"
    arrival_points="0" departure_points="0" leading_points="0" penalty="0" penalty_points="70"
    penalty_reason="Early Start" penalty_points_auto="0" penalty_reason_auto="" penalty_min_dist_points="0"
    got_time_but_not_goal_penalty="False" started_ss="2018-01-05T13:40:00+11:00" ss_time_dec_hours="0"
    ts="2019-11-27T10:43:05+13:00" real_distance="95.03" last_distance="-1.79769313486232E+305"
    last_altitude_above_goal="0" altitude_bonus_seconds="0" altitude_bonus_time="00:00:00"
    altitude_at_ess="0" scored_ss_time="00:00:00" landed_before_stop="False" />
</FsParticipant

**Comment by philderbeast** [comment]
Showing that flare-timing picked up these values:

![Screen Shot 2020-12-26 at 9 27 33 PM](https://user-images.githubusercontent.com/633283/103162663-61ccbd00-47c1-11eb-86eb-09ac6eba0fc9.png)
![Screen Shot 2020-12-26 at 9 27 47 PM](https://user-images.githubusercontent.com/633283/103162664-62655380-47c1-11eb-8e06-dca440bd30b5.png)

Note that when picking up the distance awarded by the scorer, I take it as a ratio of task distance.

**Comment by biuti** [comment]
I pushed an update for the function (https://github.com/FAI-CIVL/FAI-Airscore/commit/5a25bafa9699a30e3bc98304c483483dbc897fe4)
I tested with some competitions, Forbes 2018 as well, and results seem to be accurate with official ones.
Pilots without a track with assigned distance and score are labeled as `mindist`, but this is as expected to me, as if we convert event, we don't have a track to score, so original result is kept.
I'll leave this labeled to test but I'm quite sure it is ok.
